staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/v2alpha1/types.go

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
This looks like a pre-existing issue for the other KMS proto packages, but there should be OWNERS files in the API-facing KMS packages that look something like staging/src/k8s.io/kubelet/pkg/apis/credentialprovider/OWNERS, to make sure changes go through API review

```
# See the OWNERS docs at https://go.k8s.io/owners

# Disable inheritance as this is an api owners file
options:
  no_parent_owners: true
approvers:
  - api-approvers
reviewers:
  - sig-auth-api-reviewers
```

Reply:
Added in https://github.com/kubernetes/kubernetes/pull/111126/commits/9349f688f787ccbfafbd16f24b62c0f08c44f2aa